test/e2e/framework/util.go

```diff
@@ -5075,3 +5076,20 @@ func GetMasterAndWorkerNodesOrDie(c *client.Client) (sets.String, *api.NodeList)
 	}
 	return masters, nodes
 }
+
+func CreateFileForGoBinData(gobindataPath, outputFilename string) error {
+	data := ReadOrDie(gobindataPath)
+	if len(data) == 0 {
+		return fmt.Errorf("Failed to read gobindata from %v", gobindataPath)
+	}
+	fullPath := filepath.Join(TestContext.OutputDir, outputFilename)
+	err := os.MkdirAll(filepath.Dir(fullPath), 0777)
```

Comment:
is `0755` enough? (i am ok with leaving it as `0777`, but just asking to learn if we needed it)